pyproject.toml

```diff
@@ -6,8 +6,14 @@ build-backend = "setuptools.build_meta"
 testpaths = ["pins"]
 addopts = "--doctest-modules"
 doctest_optionflags = "NORMALIZE_WHITESPACE"
-markers = [
-    "skip_on_github: skip this test if running on github",
-]
+markers = ["skip_on_github: skip this test if running on github"]
 
 [tool.setuptools_scm]
+
+[tool.ruff]
+line-length = 90
+extend-exclude = ["docs"]
+
+[tool.ruff.lint]
```

Comment:
these are good defaults to have parity on what was already there! it looks like `ruff` has lots of options to expand as well.